### PR TITLE
Initial repo skeleton

### PR DESCRIPTION
**Pull request by swade1987**
Contains the following:

1. Main README.md with community meeting schedule and working group list.
2. Working group directories with skeleton main README.md underneath each.
3. The `integrations` directory is fully baked (worked with @ZackButcher and @geeknoid to complete)

**Note**

The working group leads will need to complete their main readme files using the Integrations directory as an example.

### CODE REVIEW

**Comment by swade1987** [comment]
I signed it!

Regards

Steve

_________

*Steve Wade*

steven@stevenwade.co.uk
@swade1987 <https://twitter.com/swade1987>
http://www.stevenwade.co.uk/

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+# Table of Contents
+
+1. [Community Meeting](#community-meeting)
+2. [Working Groups](#working-groups)
+3. [How can I help?](#how-can-i-help-)
+4. [General Developer Communication Information](communication.md)
+
+## Community Meeting
+
+We have PUBLIC and RECORDED [bi-weekly community meetings](https://zoom.us/j/986657835) every Thursday at 11am US Pacific Time.
```
Comment (ZackButcher) [inline README.md]: Not sure if the meeting ID is stable or not, I'll ask April and circle back. We should link to the recorded meetings too, but again I'll verify where they are and send a PR.

**Comment by ldemailly** [comment]
should we duplicate the WG list from istio/istio ?

**Comment by ZackButcher** [comment]
istio/istio should probably be updated to point here, this repo feels like a better source of truth for that info.

**Comment by geeknoid** [comment]
Agreed. We should eliminate redundancy between istio/istio,
istio/community, and istio.io. Ideally, the truth is kept in one place only
and we have pointers between the docs as necessary.

**Comment by ldemailly** [comment]
(random) whoever as access to the settings can you make it so it's a squash and merge next time ?

File: README.md (hunk 1 above)
Comment (thisisnotapril) [inline README.md]: Recordings go on our YouTube channel - https://www.youtube.com/channel/UC-zVlo1F3mUbExQ96fABWcQ

**Comment by geeknoid** [comment]
I've gone through and updated the repo settings to match all our other
repos. As part of this, I created the Community Hackers and Community
Admins teams and prepropulated a few names in those teams.

**Comment by ldemailly** [comment]
thanks @geeknoid - hope you put April in there - can you put me too as admin/backup

**Pull request merged**